src/App.js

```diff
@@ -39,6 +39,7 @@ function Viewer({ widgetSrc, code, initialProps }) {
   useEffect(() => {
```

Comment:
RPC solution is to replace the need for this useEffect

Currently, this useEffect fetches the redirectMap served from bos-workspace via a url in local storage